package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
```

Comment:
- [ OPTIONAL ] Kindly write a proper description for your title 👍🏾

Reply:
Noted as well